src/types/index.ts

```diff
@@ -19,10 +75,7 @@ type StopGroupGpuid = Gpuid;
 
 type StopClusterGpuid = Gpuid;
 
-enum GroundPlaceType {
-  Group = 'group',
-  Cluster = 'cluster',
-}
+type GroundPlaceType = 'group' | 'cluster';
```

Comment:
Enum

Reply:
I've tried to type this, but for some reasons TS is complain about it, I'll look at this in depth to understand why